lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -62,7 +62,7 @@
 
```

Comment:
The line above (which I cannot mark...) should be updated as well?

Currently:
```
 ax.axis["bottom"].major_ticks.set_tick_out(True)
```